src/components/utilities/Input.js

```diff
@@ -136,12 +154,30 @@ class Input extends Component {
             value={this.props.value}
             onChange={this.handleChange}
             placeholder={placeholder}
-            autoComplete={autoComplete ? null : 'new-password'}
+            autoComplete={autoComplete ? 'on' : 'off'}
```

Comment:
Maybe use a boolean here instead ?

Reply:
the variable `autoComplete` is a boolean
and the attribute `autoComplete` (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) is a string, and can be on, off or a lot of other stuff.

Reply:
Yep my bad, just saw this